### ISSUE TITLE
Redesign: Landing page feedback

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#4090]
Hello, I was looking at the https://beta.kodadot.xyz/?redesign=true and found a few things that need to be changed. I will mention everything I see as I open it. Some of these you may already know about, but I will try to highlight them all here.

## Hero section
- There should not be that much margin between "Discover, Collect And Sell" and "Kusama NFTs" It's maybe something like 10 or fewer px.
-  side graphics collapses into middle content. 
![image](https://user-images.githubusercontent.com/90852205/194551123-bc781e30-5a9f-493b-a31c-8f226075a63f.png)
- Links under the search bar when clicked should direct the user to explore the page with the chosen network
![image](https://user-images.githubusercontent.com/90852205/196035987-0cebb8d9-61ce-46ca-a437-67d542e2e78c.png)
Does the network in brackets have to be here? It's not looking great how it's not aligned vertically (or has the same width) as a search bar. 

## search bar/results
current status: 
![image](https://user-images.githubusercontent.com/90852205/196338758-9cc29687-3538-40c8-b47b-068cc1ddfadc.png)
the design: 
![image](https://user-images.githubusercontent.com/90852205/196338831-0b0b44c3-2ad1-41fb-967b-9f639e561658.png)
on page https://beta.kodadot.xyz/?page=1&redesign=true&search= there are no trading results when focusing on the search bar or pressing "K" 
- Hover effect missing on all cards
- White mode hover effect: accent light 2
- dark mode hover effect: `#363234`
demo: 

https://user-images.githubusercontent.com/90852205/196341528-db33d7f7-9cc5-4180-81ad-7ec1e7b8b3be.mp4

## Missing info on collections
![image](https://user-images.githubusercontent.com/90852205/196340391-e7328dc0-0643-436a-9467-fa0f0af7c22e.png)
- floor and units

## Selection
![image](https://user-images.githubusercontent.com/90852205/196340555-bacaa2b3-1e22-4620-be95-ea682b07fe9f.png)
how it should look like: 
![image](https://user-images.githubusercontent.com/90852205/196340607-b0e09f88-ff33-4858-af80-d2c94ece4c06.png)
- there is no hover effect on the text. It's not the nav section. When active - bold text

"see all"
- it should have only the hover effect - bold, not changing any color. 
![image](https://user-images.githubusercontent.com/90852205/196341312-6dc4bc07-b752-40e3-a3d2-e62412ace0b2.png)

## Spotlight
- All of the cards in the carousel, including single nft cards in Newest lists, Latest sales, etc. should overflow over the edges of the content. They should scale up/down to show only 4 in the case of the spotlight and single nfts. Once it overflows, it should change its opacity to 30% and mode to luminosity. That means we see the content, but it's clear we can scroll. See the picture below.
now:  
![image](https://user-images.githubusercontent.com/90852205/194552746-1b20b4b7-7f5a-42b5-92aa-662300145b77.png)
in the design file: 
![image](https://user-images.githubusercontent.com/90852205/194552801-738e0389-e0dd-48c9-9407-683b1eee4b5e.png)
- heading should be white and also respect the font sizing of h2 - in this case, it's 39px, but that's a minor problem. 
- also the images in the spotlight card should have a stroke of 1px. Like this: 
![image](https://user-images.githubusercontent.com/90852205/194553082-0807bce2-846d-4690-8273-4e34ab09e133.png)
- Then there's a slight problem with the drop shadow effect, but I think that will be solved with problem n. 1.
![image](https://user-images.githubusercontent.com/90852205/194553348-4e6c9691-c770-48dc-a4f2-54492d455956.png)

## Top collections
- On the bigger screens, there are 3 columns next to each other of top/trending collection cards. 
![image](https://user-images.githubusercontent.com/90852205/194553600-4ed211e7-035c-47fb-80bb-8eb957febc65.png)
- In the design, there are only 2. 
- content inside card: 
![image](https://user-images.githubusercontent.com/90852205/194553736-0dafad53-e8b2-4c1b-9681-746695c64d0a.png)
now it's missing network, USD volume, and % change. 
![image](https://user-images.githubusercontent.com/90852205/194553971-c72a8416-06a4-49df-bd88-192d015c3fa5.png)
- missing filters for the time period and network. 
- "see more link" should be on hover similarly to the nav bar/footer links. 

## NFT carousel section
- Same issue as with the spotlight carousel.
> All of the cards in the carousel, including single nft cards in Newest lists, Latest sales, etc. should overflow over the edges of the content. They should scale up/down to show only 4 in the case of the spotlight and single nfts. Once it overflows, it should change its opacity to 30% and mode to luminosity. That means we see the content, but it's clear we can scroll. See the picture below.
![image](https://user-images.githubusercontent.com/90852205/194555142-cb1928ca-b6fd-4491-ad70-09fac5302ffb.png)

## Footer
- We already discussed this one, but icons for socials in dark mode should have different hover effect. Icons should change to dark when hovering. See the picture below. 
![image](https://user-images.githubusercontent.com/90852205/194555684-f90c3ea6-fefe-4b82-a40f-dab616702ceb.png)
- Links have hover state - bold

## Cards
![image](https://user-images.githubusercontent.com/90852205/196035695-278063a9-17c8-4843-99c8-405e05d947fe.png)
the card breaks when there is no collection/nft name. Also, In this case, it would be great to make max name length and then only add 3 dots behind if it is over.
And I think that's it. If I see any more, I'll comment on the issue. If you have any questions, ping me anytime! 

## Ref
- #3844

**Comment by yangwao** [comment]
Any takers? I guess this could be made in parallel 👀

**Comment by preschian** [comment]
> in the design file: 
![image](https://user-images.githubusercontent.com/90852205/194552801-738e0389-e0dd-48c9-9407-683b1eee4b5e.png)

can someone fix this carousel style? I'm not really good with CSS stuff, haha 🙈

**Comment by roiLeo** [comment]
I think we should open separate issue for each component change

Regarding TopCollection component: 
- I find it better to show 3 column on large/wide screen
- we don't have % change data yet
- some for filters, we need to improve `Series` resolver (ref kodadot/rubick#26)
- noted! will change hover link

**Comment by prachi00** [comment]
Just to keep track of these in separate issues: 

- [ ] Heros
- [ ] Search Bar/results - #4136
- [ ] Spotlight
- [ ] Top collections
- [ ] NFT carousel section
- [ ] Footer
- [ ] Cards

**Comment by prachi00** [comment]
@exezbcz whats wrong with the hero section? seems fine to me on 14 inch, is it an issue on smaller screens?

**Comment by exezbcz** [comment]
its a problem with the smaller screens. On the resolution 1200px and lower the graphics collapse into the text. 
![image](https://user-images.githubusercontent.com/90852205/197339207-aad6cdcc-ea18-4385-8b67-ace7e64042d3.png)


Also, the graphics aren't aligned vertically with the hero container. 
![image](https://user-images.githubusercontent.com/90852205/197339342-69513530-47ef-430f-b398-a95235b5e9b4.png)


Then there is this: 
>There should not be that much margin between "Discover, Collect And Sell" and "Kusama NFTs" It's maybe something like 10 or fewer px.

**Comment by exezbcz** [comment]
Also, could someone please fix the .body color, which is currently set to something with a grey tone? It's not true white as it should be. Thanks!

**Comment by prachi00** [comment]
> .body

fixed in #4138

**Comment by prachi00** [comment]
> Also, could someone please fix the .body color, which is currently set to something with a grey tone? It's not true white as it should be. Thanks!

fixed in #4138

**Comment by yangwao** [comment]
Is there anything left in this issue that needs to be done?

**Comment by exezbcz** [comment]
> Is there anything left in this issue that needs to be done?

Other than the data on the collection cards, it's finished.

**Comment by yangwao** [comment]
- #4263 

<img src="https://media1.giphy.com/media/65WGIINB4VfA1JuXCH/giphy.gif"/>
